test/cpp/qps/worker.cc

```diff
@@ -34,6 +34,9 @@ DEFINE_int32(driver_port, 0, "Port for communication with driver");
 DEFINE_int32(server_port, 0, "Port for operation as a server");
 DEFINE_string(credential_type, grpc::testing::kInsecureCredentialsType,
               "Credential type for communication with driver");
+DEFINE_string(
+    driver_port_result_path, "",
+    "Path for the file to have a driver port. (Useful when driver_port is 0)");
```

Comment:
This comment should further say that this mechanism only works when the driver is on the same machine as the workers and instantiates the workers itself. This is the use case for the unit test versions of the QPS tests but not the actual benchmark versions. As an alternative, have you considered passing this information from the driver to the worker as a new RPC instead of through a file, since the workers are servers?

Reply:
Practically this parameter is limited on the same machine but it's still possible if the path is accessible from both machines but I can add more context for that and I don't think I understand your suggestion, how do workers pass the info to the driver?